src/effects/native/balanceeffect.cpp

```diff
@@ -1,7 +1,11 @@
 #include "balanceeffect.h"
 
-namespace {
+#include "util/defs.h"
 
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
Why does this go so high? If this is only intended for bypassing bass frequencies, the maximum should be much lower. With a narrower range for the knob it will be easier to select the select cutoff for the bass.